explorer.js

```diff
@@ -149,6 +149,8 @@ function SharedService($rootScope) {
 		// AWS.config.update(settings.cred);
 		// AWS.config.update({ region: settings.region });
 		AWS.config.update(Object.assign(settings.cred, { region: settings.region }));
+		AWS.config.update({ correctClockSkew: true });
+		AWS.config.correctClockSkew = true;
```

Comment:
Do we really need both? Isn't `AWS.config.update({ correctClockSkew: true })` enough?

Reply:
the doc about it is really not clear, do not know if it's an alias on it do thing differently, both are listed in a different way, I could prefer to cover all possible cases which could happen in different scenarii (update seems to point to previous usage of config )
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html

Reply:
> the doc about it is really not clear, do not know if it's an alias on it do thing differently, both are listed in a different way, I could prefer to cover all possible cases which could happen in different scenarii (update seems to point to previous usage of config )
> https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html

I think `config.update` is just an alias, if you only do `AWS.config.update({ correctClockSkew: true });` and then do `console.log(AWS.config);` you will notice that AWS.config has a new `correctClockSkew` property set to `true`